csrankings.csv

```diff
@@ -6634,6 +6662,7 @@ Jes Frellsen,IT University of Copenhagen,https://pure.itu.dk/portal/en/persons/j
 Jesper Bengtson,IT University of Copenhagen,http://www.itu.dk/people/jebe,NOSCHOLARPAGE
 Jesper Buus Nielsen,Aarhus University,http://www.cs.au.dk/~jbn,PoDj4lsAAAAJ
 Jesper Kjeldskov,Aalborg University,http://people.cs.aau.dk/~jesper,pD3nfeAAAAAJ
+Jesper Larsson Träff,TU Wien,https://par.tuwien.ac.at/~traff/,NOSCHOLARPAGE

```

Comment:
Does not work.

Reply:
The link seems to be temporarily suspended. I have emailed the team to get resolved. But that is the official website of his research group.